### ISSUE TITLE
Nominators Data on staking/target needs explination

### ISSUE THREAD

**Issue by NukeManDan** [polkadot-js/apps#4895]
![image](https://user-images.githubusercontent.com/35669742/111924630-2b72a700-8a6b-11eb-8d01-ebfe82db3127.png)

under the "nominators" column, there are two reported numbers. It is unclear to a user what these correspond to.

> no.1 is the current nominator number for this validator no.2 is the unique number of nominators total ever backing this validator historically

This is what _I think_ these correspond to but am not sure. 

Ideally having the columns be uniquely titled correctly, or at least a `?` tooltip to over over that shows more of what these numbers are is needed.

**Comment by MCFlatliner** [comment]
Agree 100% . lots of questions arev askeed about this and is easily resolved like this

**Comment by jacogr** [comment]
Tooltips cannot be used, should actually be removed everywhere since it kills the pages on large lists, like here. 

- active selected via phragmen
- total nominating

As everywhere else, there is not much hand-holding the UI provides. The focus atm is all on functionality (this means parachains, crowdloans, auctions, gilts, lotteries) & optimizations (e.g. staking speed is top-priority) as opposed to handholding and it will be for a while, but will get there eventually.

TL:DR Nothing that will be done here for quite some time, but re-layout may make it in since this is due a bit of a re-look to optimize on this one as well.

**Comment by jmhickman** [comment]
Just to add something to the thread; as a user I found the ambiguously-labeled column confusing. I was curious about this line
> Tooltips cannot be used, should actually be removed everywhere since it kills the pages on large lists, like here.

I'm not a front-end developer, so I'm confused as to how loading a tooltip once on page load is going to affect performance in a meaningful way. In normal use, waiting for the data to populate from the RPC/API calls takes (sometimes) tens of seconds, so surely some static text is essentially free in comparison? What am I missing/not understanding correctly?

**Comment by jacogr** [comment]
This issue has been open for 21 days with no activity and is not labelled as an enhancement. It will be closed in 7 days.
